mlflow/server/js/.eslintrc.js

```diff
@@ -0,0 +1,451 @@
+module.exports = {
```

Comment:
... this is an intense config file. Is it based on something?

Reply:
Yes, it's based on Databricks' internal one. We could also get rid of all the customization, but it seemed sensible overall and it might catch things that we haven't anticipated as problems in this project yet.